etc/kernelspecs/python_tf_docker/kernel.json

```diff
@@ -1,6 +1,6 @@
 {
   "language": "python",
-  "display_name": "Python on Docker with Tensorflow",
+  "display_name": "Python on Docker with Tensorflow 2.x",
```

Comment:
I'd really prefer we not "go here" as this introduces 4 file updates for each major version increment and implies we DO support multiple versions.  In addition, the display-name doesn't necessarily correspond to what's in the image and folks are free to update both the image and display_name as they wish.